Dockerfile

```diff
@@ -108,8 +108,8 @@ ENV JB_JAVA8_HOME /usr/lib/jvm/${JB_JAVA8_VERSION}
 ## echo "Run export JAVA_HOME=\"/usr/lib/jvm/${JB_JAVA8_VERSION}/\" PATH=\"/usr/lib/jvm/${JB_JAVA8_VERSION}/bin:$PATH\" to select this jdk"
 
 ## install JetBrains JDK 11
-ENV JB_JAVA11_VERSION jbrsdk-11_0_3-linux-x64-b304.39
-RUN wget --progress=dot:giga -O /tmp/${JB_JAVA11_VERSION}.tar.gz https://bintray.com/jetbrains/intellij-jbr/download_file?file_path=${JB_JAVA11_VERSION}.tar.gz \
+ENV JB_JAVA11_VERSION 11_0_9-b944.49
+RUN wget --progress=dot:giga -O /tmp/${JB_JAVA11_VERSION}.tar.gz https://projects.itemis.de/nexus/service/local/repositories/mbeddr/content/com/jetbrains/jdk/jbrsdk/${JB_JAVA11_VERSION}/jbrsdk-${JB_JAVA11_VERSION}-linux-x64.tgz \
```

Comment:
What is 'service/local' in this path? I'm used to using `https://projects.itemis.de/nexus/content/repositories/mbeddr/com/jetbrains/...` instead.

Reply:
It seems that the UI spits out a different path from when navigating there from the repository path. Strange.

I changed it.

<img width="863" alt="grafik" src="https://user-images.githubusercontent.com/569215/105879893-03cf0780-6003-11eb-810e-3bc5f6734ab9.png">